examples/vpc-scenario-2/main.tf

```diff
@@ -207,7 +219,13 @@ END_INIT
 #    Name = "${var.name}-bastion"
 #  }
 #}
+
 output "elb_dns" {
   value       = "${aws_elb.web.dns_name}"
   description = "make the ELB accessible on the outside"
 }
+
+resource "aws_key_pair" "main" {
+  key_name   = "${var.name}"
+  public_key = "${file(var.ssh_pubkey)}"
+}
```

Comment:
I don't see a reason why this resource should be moved to the bottom. Without a purpose we can articulate, this type of churn is discouraged.